x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form.tsx

```diff
@@ -111,6 +113,44 @@ export function SloEditForm({ slo }: Props) {
   const { mutateAsync: createSlo, isLoading: isCreateSloLoading } = useCreateSlo();
   const { mutateAsync: updateSlo, isLoading: isUpdateSloLoading } = useUpdateSlo();
 
+  const createProcessedValues = () => {
+    const values = getValues();
+    if (isEditMode) {
+      return transformValuesToUpdateSLOInput(values);
+    }
+    return transformValuesToCreateSLOInput(values);
+  };
```

Comment:
For the use case of copying the SLO JSON data, I think we can use the `transformValuesToCreateSLOInput(values);` regardless of the `isEditMode`. Moreover, they are technically identical at the moment, only the returned type differs to please react query.

Reply:
Makes sense... I'll remove the extra function.